### ISSUE TITLE
GeoSeries import fails with pandas 1.1.0

### ISSUE THREAD

**Issue by BerndSchrooten** [geopandas/geopandas#1559]
GeoSeries import started to fail unexpectedly on new installs. Turns out pandas released a new version 1.1.0 a couple days ago, locking pandas to 1.0.5 solved the issue.

```
/usr/local/lib/python3.7/dist-packages/geopandas/__init__.py:1: in <module>
    from geopandas.geoseries import GeoSeries
/usr/local/lib/python3.7/dist-packages/geopandas/geoseries.py:13: in <module>
    from geopandas.base import GeoPandasBase, _unary_op, _CoordinateIndexer
/usr/local/lib/python3.7/dist-packages/geopandas/base.py:6: in <module>
    from pandas.core.indexing import _NDFrameIndexer
E   ImportError: cannot import name '_NDFrameIndexer' from 'pandas.core.indexing'
```

**Comment by jorisvandenbossche** [comment]
What version of GeoPandas are you using? Normally, this should already be fixed starting from GeoPandas 0.6 (see https://github.com/geopandas/geopandas/issues/1042)

**Comment by BerndSchrooten** [comment]
You are right!
